src/Selenium2Library/base.py

```diff
@@ -0,0 +1,100 @@
+from robot.api import logger
+from selenium.webdriver.remote.webelement import WebElement
+
+from .locators.elementfinder import ElementFinder
+
+
+LOG_LEVELS = ['TRACE', 'DEBUG', 'INFO', 'WARN', 'ERROR']
+
+
+class Base(object):
+
+    def __init__(self, ctx):
+        self.ctx = ctx
+        self.element_finder = ElementFinder()
+        self._speed_in_secs = 0.0
+        self._timeout_in_secs = 5.0
+        self._implicit_wait_in_secs = 5.0
```

Comment:
No all instances must have the same value, that was the way it worked before the refactoring. Those timers where in https://github.com/robotframework/Selenium2Library/blob/master/src/Selenium2Library/keywords/browsermanagement.py#L45 but I moved them to Base. Other place that I was thinking to put them was the library `__init__.py` but can not anymore recall why I did actually put the timers here. Changed to not the instance specific but perhaps `__init__` would be better place after all.